build.gradle.kts

```diff
@@ -112,6 +114,7 @@ val benchmark by task<integration.Benchmark> {
 
 // -- IntelliJ IDEA configuration --------------------------------------
 idea {
+    (this as IdeaModel)
```

Comment:
Why is this now necessary?

Reply:
This is caused by a change in `gradle/gradle`.
That's a temporary measure until https://github.com/gradle/gradle/pull/5644 is merged.